spring/src/test/java/io/cloudevents/spring/messaging/CloudEventMessageUtilsTests.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2020-Present The CloudEvents Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cloudevents.spring.messaging;
+
+import java.net.URI;
+import java.util.Map;
+
+import io.cloudevents.core.v1.CloudEventBuilder;
+import io.cloudevents.spring.core.CloudEventAttributeUtils;
+import io.cloudevents.spring.core.MutableCloudEventAttributes;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.converter.MappingJackson2MessageConverter;
+import org.springframework.messaging.support.MessageBuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ *
+ * @author Oleg Zhurakousky
+ *
+ */
+public class CloudEventMessageUtilsTests {
+
+	String payloadWithHttpPrefix = "{\n" +
+	        "    \"ce-specversion\" : \"1.0\",\n" +
+	        "    \"ce-type\" : \"org.springframework\",\n" +
+	        "    \"ce-source\" : \"https://spring.io/\",\n" +
+	        "    \"ce-id\" : \"A234-1234-1234\",\n" +
+	        "    \"ce-datacontenttype\" : \"application/json\",\n" +
+	        "    \"ce-data\" : {\n" +
+	        "        \"version\" : \"1.0\",\n" +
+	        "        \"releaseName\" : \"Spring Framework\",\n" +
+	        "        \"releaseDate\" : \"24-03-2004\"\n" +
+	        "    }\n" +
+	        "}";
```

Comment:
More than hardcoding these, how about putting them in test resources?

Reply:
Generally I like it that way for tests since it keeps everything in single place. That said I am ok to moving it if you feel strongly about it.